Moonscraper Chart Editor/Assets/Scripts/Game/Misc/SongValidate.cs

```diff
@@ -194,6 +194,50 @@ static string CheckForErrorsCloneHero(Song song, ValidationParameters validation
             }
         }
 
+        foreach (var instrument in EnumX<Song.Instrument>.Values)
+        {
+            if (instrument != Song.Instrument.Drums && instrument != Song.Instrument.Unrecognised)
+            {
+                Chart lastChart = null;
+                foreach (var difficulty in EnumX<Song.Difficulty>.Values)
+                {
+                    var chart = song.GetChart(instrument, difficulty);
+
+                    // Skip if chart is same as last
+                    if (chart == lastChart)
+                    {
+                        continue;
+                    }
+
+                    lastChart = chart;
```

Comment:
What is this actually intending to skip? I'm struggling to see when this would actually be hit.